paltax/InputConfigs/input_config.py

```diff
@@ -103,13 +103,23 @@ def get_config():
         'exposure_time': 1024, 'num_exposures': 2.0, 'sky_brightness': 22,
         'magnitude_zero_point': 25, 'read_noise': 3.0
     }
-    cosmos_path = str(pathlib.Path(__file__).parent.parent.parent)
-    cosmos_path += '/datasets/cosmos/cosmos_galaxies_train.npz'
+
+    root_path = str(pathlib.Path(__file__).parent.parent.parent)
+    cosmos_path = root_path + '/datasets/weighted_catalog.h5'
+    
+    hdf5_file = h5py.File(cosmos_path, 'r')
```

Comment:
Here an in other init files, use with to open the hdf5 file and automatically close it once the weights are extracted